components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -142,6 +170,37 @@ export default function Preferences() {
                         </p>
                     </div>
                 </div>
+
+                <h3 className="mt-12">Timeout </h3>
+                <p className="text-base text-gray-500 dark:text-gray-400">Customize timeout setting for workspace.</p>
```

Comment:
**question** What do you think of replicating here what is in the [relevant docs](https://www.gitpod.io/docs/configure/workspaces/workspace-lifecycle#workspace-inactivity:~:text=By%20default%2C%20workspaces%20stop%20following%2030%20minutes%20without%20user%20input%20(e.g.%20keystrokes%20or%20terminal%20input%20commands).%20You%20can%20increase%20the%20workspace%20timeout%20up%20to%20a%20maximum%20of%2024%20hours.) for accuracy? Adding this inline, a break line will probably be needed because of the linter.

```suggestion
                <p className="text-base text-gray-500 dark:text-gray-400">By default, workspaces stop following 30 minutes without user input (e.g. keystrokes or terminal input commands). You can increase the workspace timeout up to a maximum of 24 hours.</p>
```